apps/spruce/src/pages/version/TestAnalysis/index.tsx

```diff
@@ -180,10 +177,11 @@ const TestAnalysis: React.FC<TestAnalysisProps> = ({ versionId }) => {
           {hasResults && (
             <FilterSubheaderContainer>
               <Body weight="medium">
-                {totalFilteredTestCount}/{totalTestCount} Filtered Failed{" "}
+                {totalFilteredTestCount}/{totalTestCount} Failed{" "}
                 {pluralize("Test", totalFilteredTestCount)}
               </Body>
               <Button
+                disabled={hasFiltersApplied}
```

Comment:
Shouldn't disable if there is a filter!
```suggestion
                disabled={!hasFiltersApplied}
```

Reply:
Bump